freqtrade/constants.py

```diff
@@ -432,6 +432,8 @@
 
 CANCEL_REASON = {
     "TIMEOUT": "cancelled due to timeout",
+    "ENTRYPRICECHANGED": "Custom entry price changed",
+    "EXITPRICECHANGED": "Custom exit price changed",
```

Comment:
I'd think this is also still part of the "timeout logic" ... ;)

Reply:
Yes I will remove that